<launch>
    <group ns="sensorUV_specs">
        <rosparam command="load" file="$(find mir100_germgrid)/config/sensorUV.yaml" />    
    </group>
    
    <node pkg="mir100_germgrid" type="germgrid" name="irradiation_map"/>          
</launch>
